lib/jobsub_api.py

```diff
@@ -74,3 +90,519 @@ def jobsub_call(argv: List[str], return_output: bool = False) -> Optional[str]:
         print(f"Excepion in jobsub_call({argv})")
         raise
     return res
+
+
+# =-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+
+def optfix(s: Optional[str]) -> str:
```

Comment:
Rather than have this function wrapping every `jobsub_call` invocation, perhaps it's better to have `jobsub_call` only return a string.  Looking at that function, it looks like the only reason it needs to be `Optional[str]` is because you initialize `res` to `None`.  Maybe if that's `""`, then you can always return a string, and then this function won't be necessary.

Reply:
Changed per shreyb's suggestion.   Initialized to "" , and got rid of optfix() and Optional import. [upper_api 44fd798]